Automattic/wp-calypso
# Remove mentions of `shouldShowOfferResetFlow` in source code

### Changes proposed in this Pull Request

This PR removes the conditions that check if the offer reset flow is enabled. It's now always enabled.

### Testing instructions

- Do a code review of the updated files
- Check there's no instance left of `shouldShowOfferResetFlow` in the codebase
- Visit the plans page and check that it looks as expected, and that there's no error in the browser's console
